SphereEversion/Global/Immersion.lean

```diff
@@ -286,21 +286,21 @@ variable {𝕜 : Type*} [NontriviallyNormedField 𝕜]
   {P : Type*} [TopologicalSpace P] [ChartedSpace HP P]
 
 -- move to Mathlib.Geometry.Manifold.ContMDiff.Product
-lemma ContMDiff.inr {n : ℕ∞} (x : M) :
+lemma ContMDiff.inr' {n : ℕ∞} (x : M) :
```

Comment:
This is pre-existing: in my opinion, `ContMDiff.inr` is a bad name, since `ContMDiff.inr` already exists in mathlib and actually speaks about `Sum.inr`.

Reply:
In other words, I would support moving to the prod naming... but asking on zulip is also a fine idea.

Reply:
In light of the [zulip discussion](https://leanprover.zulipchat.com/#narrow/channel/303200-sphere-eversion/topic/Mathlib.20bumps), can you switch to the prod naming (at least)? Thanks! I'll merge with that applied.